src/utils.js

```diff
@@ -11,6 +11,12 @@ const exec = async (command) => {
       command,
       { ...process.env },
       (error, stdout, stderr) => {
+        if (!process.stdout.isTTY) {
+          const controlSequence =
+            /(?<csi_char>(\\u001b|\u001b)\[)(?<sgr_bytes>(\d*;?)+)(?<sgr_term>m)/g; // eslint-disable-line no-control-regex
```

Comment:
```suggestion
            /(?<csi_char>\u001b\[)(?<sgr_bytes>(\d*;?)+)(?<sgr_term>m)/g; // eslint-disable-line no-control-regex
```

Shouldn't we only match the `␛` character?

Reply:
It never happened in my testing but I threw in the literal form just incase in came up. It never did with what I tested to taking it out works for me.

Reply:
I believe that, if it came escaped, we should include it in the sanitized output, lest corrupting the data.

One thing is removing escape secuences and a very different one is removing printable characters that don't belong to one of such sequences. 😈

Reply:
Off–topic, for the laughs: https://en.wikipedia.org/wiki/Constrained_writing